src/Settings/SettingsInterface.php

```diff
@@ -43,9 +43,9 @@ public function getFormClass(): string;
     /**
      * @SuppressWarnings(PHPMD.BooleanArgumentFlag)
      */
-    public function getSettingsByChannelAndLocale(?ChannelInterface $channel = null, ?string $localeCode = null, bool $withDefault = false): array;
+    public function getSettingsByChannelAndLocale(ChannelInterface $channel = null, string $localeCode = null, bool $withDefault = false): array;
 
-    public function getSettingsValuesByChannelAndLocale(?ChannelInterface $channel = null, ?string $localeCode = null): array;
+    public function getSettingsValuesByChannelAndLocale(ChannelInterface $channel = null, string $localeCode = null): array;
```

Comment:
**❔ question:** ‏Why did you remove the nullable possibility?

Reply:
oops, mistake, that was done by phpcs before I changed the rules to not have this change.